scripts/build-prep-1node.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+
+# Clone, build and prepare to start a single-node cluster.
+#
+# Usage: ./build-prep-1node.sh [-dev]
+#
+#   -dev  development mode, don't build and install rpms
+#
+set -e -o pipefail
+
+dev_mode=
+
+[[ $1 == "-dev" ]] && dev_mode='yes'
+
+which git || sudo yum install -y git
+
+[[ -d cortx-motr ]] || {
+    git clone --recurse https://github.com/Seagate/cortx-motr.git &&
+        ln -s cortx-motr motr
+}
+
+echo 'Install Motr deps...'
+sudo yum install -y epel-release # Install EPEL yum repo
+sudo yum install -y epel-release # Update to the latest version
+sudo yum install -y ansible
+
+cd motr
+sudo scripts/install-build-deps
```

Comment:
Install-build-deps is not ready for centos8 and RL 8.4.
Even if `pip3 install ipaddress` is used for `python-ipaddress`

Until that issue is fixed, it is better to have some alternative.

Reply:
I think we can work on the `install-build-deps` script issues separately, we have https://github.com/Seagate/cortx-motr/issues/1356 for that.

Reply:
Should be fixed by now at https://github.com/Seagate/cortx-motr/pull/1367.